docs/content/CONTRIBUTING.adoc

```diff
@@ -233,3 +231,16 @@ See https://docs.antora.org/antora/2.3/examples-directory/
 This allows user to download the source as attachments by providing a link directly for the code (no duplication between the actual examples and the documentation)
 
 Attachments are also possible: https://docs.antora.org/antora/2.3/attachments-directory/
+
+=== Troubleshooting sections
+https://documentation.bonitasoft.com/bonita/2021.2/bonita-overview/ui-designer-overview#_troubleshooting[Here is a good example] of what *Troubleshooting sections* should look like:
+* What is always here: 
+ ** The icon with a screen and a monkey wrench. xref:image:images/troubleshooting.png[Here is the link] to reference it
```

Comment:
broken link: please provide a link to an image in the bonita-doc repository 😸

Reply:
eh oui... ok.